src/wishlist.tm

```diff
@@ -21,7 +21,55 @@
 
   \ <hrule>
 
-  \;
+  Sometimes, one has to produce quite compacts documents, where space is key
+  (e.g.<nbsp>a short report, an exercice sheet or correction<text-dots>). One
+  very important feature you be to put figures on the left or the right of
+  the page, in a floating manner. To emulate this, tables can be used, with
+  one cell for the text and one cell for the figure, but it does not work
+  well at all with figure legends, footnotes<text-dots>
+
+  More generally, mouse-movable floating minipages would be a killer feature
+  and solve many issues at the same time.
+
+  <hrule>
+
+  An user-friendly interface to create/modify shortcuts.
+
+  <hrule>
+
+  Being able to easily create macros, at least non-parametrical ones, with a
+  single menu action. It is already possible, but the syntax not easy for
+  begginers (and non-programmers in general).
+
+  <hrule>
+
+  Being able to adjust the math typesetting parameters. For example, by
```

Comment:
What you really mean here? Can you be more precise. The design of a new feature is an important step. Should all the binary operation have adjustable spacing. I can imagine that $x<4$ (inline) or $$ x < 4$$ (display) should behave differently. What about $a<b<c<d<e$ ? Maybe what you want is just large spacing for display formulas with only one prominent inequality? Cannot you use the \eqnarray environment in this case?

Reply:
I guess there are many constants used internally for typesetting. It's would simply be a matter of modifying them with sliders, i.e. making them (or a relevant subset) accessible to the user.